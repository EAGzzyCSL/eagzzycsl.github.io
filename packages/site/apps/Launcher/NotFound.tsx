import React from 'react'

import { GetStaticPropsResult } from 'next'

import { SentimentDissatisfiedRoundedIcon } from '@/ui/icons'
import { Typography } from '@/ui/material'

import styles from './NotFound.module.scss'

const NotFound = (): JSX.Element => (
  <section className={styles.notFound}>
    <SentimentDissatisfiedRoundedIcon
      style={{
        fontWeight: 'lighter',
        fontSize: '80px',
      }}
    />
    <Typography variant='h2' component='h1'>
      404 Not Found
    </Typography>
  </section>
)

export default NotFound

export const getStaticProps = (): GetStaticPropsResult<unknown> => ({
  props: {},
})
