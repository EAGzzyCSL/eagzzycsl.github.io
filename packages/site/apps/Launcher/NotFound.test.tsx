import React from 'react'
import * as renderer from 'react-test-renderer'

import NotFound from './NotFound'

test('render NotFound', () => {
  const tree = renderer.create(<NotFound />).toJSON()
  expect(tree).toMatchSnapshot()
})
