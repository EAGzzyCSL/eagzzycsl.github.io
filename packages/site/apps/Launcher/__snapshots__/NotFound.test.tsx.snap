// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`render NotFound 1`] = `
<section
  className="_notFound_e679f1"
>
  <svg
    aria-hidden={true}
    className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1umw9bq-MuiSvgIcon-root"
    data-testid="SentimentDissatisfiedRoundedIcon"
    focusable="false"
    style={
      {
        "fontSize": "80px",
        "fontWeight": "lighter",
      }
    }
    viewBox="0 0 24 24"
  >
    <circle
      cx="15.5"
      cy="9.5"
      r="1.5"
    />
    <circle
      cx="8.5"
      cy="9.5"
      r="1.5"
    />
    <path
      d="M11.99 2C6.47 2 2 6.48 2 12s4.47 10 9.99 10C17.52 22 22 17.52 22 12S17.52 2 11.99 2M12 20c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8m0-6c-1.9 0-3.63.97-4.65 2.58-.22.35-.11.81.24 1.03s.81.11 1.03-.24c.74-1.18 2-1.88 3.38-1.88s2.64.7 3.38 1.88c.14.23.39.35.64.35.14 0 .27-.04.4-.11.35-.22.46-.68.24-1.03C15.63 14.96 13.9 14 12 14"
    />
  </svg>
  <h1
    className="MuiTypography-root MuiTypography-h2 css-o2za6n-MuiTypography-root"
    style={{}}
  >
    404 Not Found
  </h1>
</section>
`;
